setup.py

```diff
@@ -179,10 +180,17 @@ def run(self):
     author=__author__,
     author_email='info@ansible.com',
     url='https://ansible.com/',
+    project_urls={
+        'Bug Tracker': 'https://github.com/ansible/ansible/issues',
+        'Documentation': 'http://docs.ansible.com/ansible/',
+        'Source Code': 'https://github.com/ansible/ansible',
+        'Continuous Integration': 'https://app.shippable.com/github/ansible/ansible',
+    },
     license='GPLv3+',
     # Ansible will also make use of a system copy of python-six and
     # python-selectors2 if installed but use a Bundled copy if it's not.
     install_requires=install_requirements,
+    python_requires='>=2.6,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
```

Comment:
I lean towards this but am a little conflicted.  With older setuptools, these will issue UserWarnings, for instance:
```
/usr/lib64/python2.6/distutils/dist.py:266: UserWarning: Unknown distribution option: 'project_urls'
  warnings.warn(msg)
```
I think some of our users will complain but I like the additional information it will upload to pypi... I'll ask on the core slack channel and if any  of the other core team members have a problem with the tradeoff.

Reply:
We can add it conditionally, it's not a problem.

Reply:
On slack, sivel had the idea that we can check whether setuptools supports these features and only add them to the call to setup if they're available.  That sounds like a good idea to me.  I don't know of an easy way to do a feature check with setuptools so you may have to figure out the version of setuptools that each of these showed up in and do a version check instead.

Reply:
Also on slack, we've agreed that suppressing specific warning is also okay.

Reply:
I guess it's done now. Could you please check that it suppresses actual warning?